lib/presenter/subject_edit_or_delete.dart

```diff
@@ -4,6 +4,7 @@ import 'package:easy_study/model/exam_type.dart';
 import 'package:easy_study/model/priority.dart';
```

Comment:
please format all files you modified with strg + alt + L.

That is also the reason why travis fails.

Reply:
u could also right click on the lib folder inside your ide and click "Reformat Code with dartfmt..."